core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -154,6 +305,9 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
         </div>
 
       var content = summary
+      content ++= <h4>Events on Application Timeline</h4> ++
```

Comment:
Could this say "Event Timeline" and also, can you make it so that you can toggle it (see the approach in #5547)?